backend/console/console.go

```diff
@@ -35,49 +35,68 @@ import (
 type Config struct {
 	ConsoleURL  *url.URL  `help:"The public URL of the console (for CORS)." env:"FTL_CONTROLLER_CONSOLE_URL"`
 	ContentTime time.Time `help:"Time to use for console resource timestamps." default:"${timestamp=1970-01-01T00:00:00Z}"`
-	Bind        *url.URL  `help:"Socket to bind to." default:"http://127.0.0.1:8899" env:"FTL_BIND"`
 }
 
-type service struct {
+type Service struct {
 	schemaEventSource *schemaeventsource.EventSource
 	timelineClient    *timelineclient.Client
 	adminClient       admin.EnvironmentClient
 	callClient        routing.CallClient
 	buildEngineClient buildenginepbconnect.BuildEngineServiceClient
 }
 
-var _ consolepbconnect.ConsoleServiceHandler = (*service)(nil)
+var _ consolepbconnect.ConsoleServiceHandler = (*Service)(nil)
 
-func Start(ctx context.Context, config Config, eventSource *schemaeventsource.EventSource, timelineClient *timelineclient.Client, adminClient admin.EnvironmentClient, client routing.CallClient, buildEngineClient buildenginepbconnect.BuildEngineServiceClient) error {
-	logger := log.FromContext(ctx).Scope("console")
-	ctx = log.ContextWithLogger(ctx, logger)
+func New(
+	eventSource *schemaeventsource.EventSource,
+	timelineClient *timelineclient.Client,
+	adminClient admin.EnvironmentClient,
+	client routing.CallClient,
+	buildEngineClient buildenginepbconnect.BuildEngineServiceClient) *Service {
 
-	svc := &service{
+	return &Service{
 		schemaEventSource: eventSource,
 		timelineClient:    timelineClient,
 		adminClient:       adminClient,
 		callClient:        client,
 		buildEngineClient: buildEngineClient,
 	}
+}
 
-	consoleHandler, err := frontend.Server(ctx, config.ContentTime, config.Bind)
+func (svc *Service) Start(ctx context.Context,
```

Comment:
This pattern is different to the other services. I don't really mind which pattern we use, but we should settle on one rather than having multiple.

That said, this one seems more complex, as it has Start() and StartBindings()...also it's not clear at all what "StartBindings()" means.